### DISCUSSION TITLE
Container queries repo doesn't have a license

### DISCUSSION THREAD

**Discussion by Marco-Pagani** [tailwindlabs/tailwindcss#11196] [Ideas]
Not sure if this ought to be a bug issue in that repo.

[tailwindcss-container-queries](https://github.com/tailwindlabs/tailwindcss-container-queries) doesn't have a license at the moment, and I image that it's your intention to license it MIT like the main repo. Is this plugin intentionally unlicensed right now, or is that something that can be quickly added to it?

**Comment by adamwathan** [comment]
Added, thanks!
